app/modules/questionnaire/server.R

```diff
@@ -39,36 +37,36 @@ server <- function(input, output, session) {
         params <- parse_params(question$params)
 
         switch(question$q_type,
-          radioButtons = radioButtons(inputId = question$q_variable,
+          radioButtons = shiny::radioButtons(inputId = question$q_variable,
+                                             label = question$q_text,
+                                             choices = strsplit(params[["choices"]], ",")[[1]],
+                                             width = 500),
+          numeric = shiny::numericInput(inputId = question$q_variable,
+                                        label = question$q_text,
+                                        min = as.numeric(params[["min"]]),
+                                        value = 0,
+                                        width = 500),
+          slider = shiny::sliderInput(inputId = question$q_variable,
                                       label = question$q_text,
-                                      choices = strsplit(params[["choices"]], ",")[[1]],
+                                      min = as.numeric(params[["min"]]),
+                                      max = as.numeric(params[["max"]]),
+                                      value = as.numeric(params[["min"]]),
                                       width = 500),
-          numeric = numericInput(inputId = question$q_variable,
-                                 label = question$q_text,
-                                 min = as.numeric(params[["min"]]),
-                                 value = 0,
-                                 width = 500),
-          slider = sliderInput(inputId = question$q_variable,
-                               label = question$q_text,
-                               min = as.numeric(params[["min"]]),
-                               max = as.numeric(params[["max"]]),
-                               value = as.numeric(params[["min"]]),
-                               width = 500),
-          text = textInput(inputId = question$q_variable,
-                           label = question$q_text,
-                           placeholder = "Enter your hint here",
-                           value = "", width = 500)
-        )
+          text = shiny::textInput(inputId = question$q_variable,
+                                  label = question$q_text,
+                                  placeholder = "Enter your hint here",
```

Comment:
Just a minor comment here: Was there meant to be a hint? :)

Reply:
Thank you! I'll edit then merge :)